manifests/deploy/openshift/olm/bundle/4.6/meteringoperator.v4.6.0.clusterserviceversion.yaml

```diff
@@ -253,7 +253,7 @@ spec:
 
   keywords: [metering metrics reporting prometheus chargeback]
   version: "4.6.0"
-  minKubeVersion: "1.17.0"
+  minKubeVersion: "1.18.3"
```

Comment:
We're setting a different patch version here: https://github.com/kube-reporting/metering-operator/blob/e3165873ffda515bf641da2371122f4c2ae41187/charts/metering-ansible-operator/upstream-values.yaml#L50 and in one other place.

Reply:
Yeah, good catch. Updated that chart and re-made the manifests in the last force-push.